### PR TITLE
add the flow tooltip and remove async from registration check

### PR DESCRIPTION
**Pull request by xavluiz**
https://www.loom.com/share/259b68517cc443cab7e2251137b8ed0c

handling color theme change
https://www.loom.com/share/d1f5a720ee684844a126cb1cb2869d40

### CODE REVIEW

File: resources/templates/edit_settings.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                <select name="screenMode" id="screenMode" class="select">
+                    <option value="Zen" ${this.zenSelected}>Zen</option>
+                    <option value="Full Screen" ${this.fullScreenSelected}>Full Screen</option>
+                    <option value="None" ${this.noneSelected}>None</option>
+                </select>
+                <label for="screenMode">Automatically toggle a selected screen mode when I'm in flow</label>
+             </div>
+
+            <div class="buttons">
+                <button id="submit">Submit</button>
```
Comment (bjacobson26) [inline resources/templates/edit_settings.html:121]: Instead of "Submit" can this be "Save"?

**Review by bjacobson26 — APPROVED** [review]
🔥  nice work

File: src/tree/KpmProviderManager.ts [hunk 2]
```diff
@@ -178,54 +178,18 @@ export class KpmProviderManager {
 
   async getFlowTreeParents(): Promise<KpmItem[]> {
     const treeItems: KpmItem[] = [];
-    const [slackStatus, slackPresence, slackDnDInfo] = await Promise.all([getSlackStatus(), getSlackPresence(), getSlackDnDInfo()]);
 
+    const inFlowSettingsTooltip = getConfigSettingsTooltip();
+    const mdstr: MarkdownString = new MarkdownString(inFlowSettingsTooltip);
     if (!isInFlowMode()) {
-      treeItems.push(this.getActionButton("Enable flow", "Enable your code flow", "codetime.enableFlow", "paw.svg"));
+      treeItems.push(this.getActionButton("Enable flow mode", mdstr, "codetime.enableFlow", "paw.svg"));
```
Comment (mmclead) [inline src/tree/KpmProviderManager.ts:185]: Flow Mode should be capitalized, as its the title of the feature

File: package.json [hunk 3]
```diff
@@ -93,6 +89,14 @@
       {
         "command": "codetime.viewDashboard",
         "title": "Code Time: View Dashboard"
+      },
+      {
+        "command": "codetime.enableFlow",
+        "title": "Code Time: Enable flow"
```
Comment (mmclead) [inline package.json:95]: "Enable Flow Mode". to match the button

File: resources/templates/edit_settings.html (hunk 1 above)
Comment (xavluiz) [reply resources/templates/edit_settings.html:121]: yeah, totally. i'll update it.

File: src/tree/KpmProviderManager.ts (hunk 2 above)
Comment (xavluiz) [reply src/tree/KpmProviderManager.ts:185]: ok, will update. thanks

File: package.json (hunk 3 above)
Comment (xavluiz) [reply package.json:95]: ah right, thanks!

**Pull request merged**